layouts/partials/home/news.html

```diff
@@ -0,0 +1,38 @@
+<section id="news" class="my-3">
+    <div class="space-one">
+        <h3 class="container py-4 d-flex justify-content-center">
+            News
+        </h3>
+    </div>
+    <div class="container py-5">
+      <div class="">
+        <h3 class="py-4">Eröffnungsfeier</h3>
+        <p>Am Samstag, den 28.09. ab 10.00 Uhr ist es so weit. Wir öffnen die Tore! Ihr seid alle herzlich eingeladen, an einem der Programmpunkte teilzunehmen, oder einfach mal so vorbeizuschauen.</p>
+        <h4 class="py-4">Programm:</h4>
+        <h5>Coworking (oben)</h5>
+        <ul>
+          <li>11:00 - 13:00: Luftdatensensoren bauen mit Marcel Belledin</li>
+          <li>13:00 - 15:00: Kleinstteile löten ohne Lötkolben - SMD-Lötworkshop</li>
+          <li>15:00 - 16.00: Rubiks Cube lösen mit @p3nny</li>
+          <li>17:30 - 18.20: F.U.C.K Frauen* und Computer Kram Meet up</li>
+          <li>18:30 Uhr - Podcast Meetup: Erfahrungsbericht: Was hätte ich gerne am Anfang gewusst? Anna Kaffee-Sahne Podcast / Alex Freise</li>
```

Comment:
```suggestion
          <li>18:30 Uhr - Podcast Meetup: Erfahrungsbericht: Was hätte ich gerne am Anfang gewusst? Alex Freise</li>
```
Ob Anna mitmacht steht noch nicht fest.